library/private/camera_impl.h

```diff
@@ -50,6 +50,9 @@ class camera_impl : public camera
   camera& setState(const camera_state_t& state) override;
   camera_state_t getState() override;
   void getState(camera_state_t& state) override;
+  angle_deg_t getAzimuth() override;
+  angle_deg_t getYaw() override;
+  angle_deg_t getElevation() override;
```

Comment:
@snoyer maybe getCurrentElevation and such would be better ?

Reply:
or even computeElevation ?

Reply:
I changed the naming convention